### ISSUE TITLE
List Workflow managment

### ISSUE THREAD

**Issue by rachidio** [gitbrent/SpRestLib#22]
Hi,
Is there any planning to implement some methods to manage (list, start & stop) list workflow instances?

Thanks,

**Comment by gitbrent** [comment]
Hi @ra6hi9,

No, I wasn't planning on doing much with Workflows in the immediate future...

**Comment by gitbrent** [comment]
### UPDATE
SharePoint Designer-based workflows have been replaced in Microsoft365 by **Flows**.

Querying a Flow via REST involves authenticating through Microsoft Graph and providing credentials to Azure's REST API.

I'm doing some Graph work now and will update this when I have more information or examples.

Having an interface to Flows would be super valuable and I can see expending energy to make that happen.

Bottom line: I don't foresee spending any time developing Workflow queries at this point as it's almost 2019 and Designer workflows are basically legacy tech at this point...
